core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -451,7 +451,31 @@ contract PricelessPositionManager is FeePayer {
         oracle.requestPrice(priceIdentifer, requestedTime);
     }
 
-    function _getOraclePrice(uint requestedTime) internal view returns (FixedPoint.Unsigned memory) {
+    function _payFinalFee() internal {
```

Comment:
nit: maybe make this private if you don't want subclasses to use it to pay their final fees (in the case of dispute, for instance)?

Reply:
good point since `Liquidatable` shouldn't be able to call this to pay fin fees

Reply:
this is now (back to)` internal`